extensions/product-recs/content/src/main/content/jcr_root/apps/core/cif/extensions/product-recs/clientlibs/storefront-events-collector/.content.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
When removing the clientlib we have to make sure that we document this as breaking change in release notes. Otherwise the user of the react-components that do not use the new experience-platform-connector extension will run into issues.

Alternatively we could keep the clientlib and add a deprecation notice. Would this be possible?